README.md

```diff
@@ -0,0 +1,166 @@
+Patreon Wordpress is a free plugin built and maintained by Patreon. When a patron pledges, they can then head over to www.yourwordpresssite.com and click your “Connect with Patreon” button to let the site know they are a patron. When you create content on WordPress you will be given the option to limit it to only paying patrons.
+
+This plugin is maintained by Patreon and will cover the basic use cases of patronage on WordPress. Looking for additional features? Extend with premium inter-operable WordPress plugins that others have developed, also found in this directory.
+
+### FEATURES
+
+- Keep your patrons engaged on your website
+- Convert your fans to patrons by showing them how much content they’d get
+- Publish patron-only posts with the full power of WordPress
+- Option to provide an ad-free experience to patrons as an added value
+
+### Installing the Patreon WordPress Plugin
+
+1. Install the plugin
+3. Activate the plugin and then click on ‘Patreon Settings’ to view the options page. It will look something like the below, copy the ‘Redirect URI‘ into the clip board and keep for the next step.
+![](https://c5.patreon.com/external/platform/wordpress-install-ss1.jpeg)
+
+### Creating an oAuth client on Patreon
+
+1. Ensure you are logged into Patreon, using your creator account. When it comes to testing this out you will want to have two Patreon accounts, one that acts as a patron and one being the content creator. The oAuth client needs to be created on your content creating account.
```

Comment:
`Ensure you are logged into Patreon, using your creator account.` -> remove comma